client/components/credit-card/README.md

```diff
@@ -0,0 +1,4 @@
+CreditCard
+==============
+
+The `CreditCard` React component serves as a container for a selectable credit card list item, and can display a stored credit card (passed as `card` prop).
```

Comment:
I won't block on this, but let's add more detail here. See https://github.com/Automattic/wp-calypso/blob/master/client/components/button/README.md for a good example, this populates usage docs in https://wpcalypso.wordpress.com/devdocs/design/button